packages/frontend/src/hooks/useInviteLink.tsx

```diff
@@ -51,23 +53,26 @@ export const useInviteLink = (inviteCode: string) =>
 export const useCreateInviteLinkMutation = () => {
     const queryClient = useQueryClient();
     const { showToastError, showToastSuccess } = useApp();
-    return useMutation<InviteLink, ApiError>(createInviteWith3DayExpiryQuery, {
-        mutationKey: ['invite_link'],
-        onError: (error1) => {
-            const [title, ...rest] = error1.error.message.split('\n');
-            showToastError({
-                title,
-                subtitle: rest.join('\n'),
-            });
+    return useMutation<InviteLink, ApiError, string>(
```

Comment:
All I added was a `, string` in this line. The rest of the changes are formatting.